### ISSUE TITLE
 Links don't work

### ISSUE THREAD

**Issue by kaffeeringe** [ttnmapper/documentation#8]
The FAQ says:

> **Can I integrate your data into my map?**
> **Can I download my data which I uploaded to you?**
> **Public API access.**
> The GEOJSON files used to draw the TTN Mapper maps are publicly available at http://ttnmapper.org/geojson.
> Data is sometimes dumped into CSV files. The dumps are available at http://ttnmapper.org/dumps. If you want newer data than what > there are dumps of, please contact me.
> No public API access is given to the raw data. This is due to the amount of data and number of possible queries that won’t be possible > to handle on the server. Please use the dumps and your own database.

The links don't work. How can I get a data dump?

**Comment by jpmeijers** [comment]
Yes indeed, those links dropped somewhere along the line as the system got rewritten. It's impractical to do periodic dumps of the data due to the sheer size of the database. For now please use the [contact form](https://docs.ttnmapper.org/support-project.html#contact) to request the specific subset of data your require.

**Comment by mirad** [comment]
I would also like to have a snapshot of the geojson or csv. I tried the contact form. Nobody replied. Any other ideas?
